skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +355,136 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
```

Comment:
Right, all valid tests are using default tau, alpha, theta so test that valid non-default values can be used. Otherwise we wouldn't know if those values were respected by the function or not.